### PR TITLE
Elu specification refactoring

### PR DESCRIPTION
**Pull request by ggalieroc**
Elu specification refactored.

Details:
- Extended detailed description
- Modified the mathematical formula describing the op
- Added type of tensors

Ticket: 46074

### CODE REVIEW

**Review by lazarevevgeny — APPROVED** [review]

File: docs/ops/activation/Elu_1.md [hunk 1]
```diff
@@ -8,29 +8,58 @@
 
 **Detailed Description**
 
-For each element from the input tensor calculates corresponding
-element in the output tensor with the following formula:
+*Elu* operation is introduced in this [article](https://arxiv.org/abs/1511.07289v3).
+It performs element-wise activation function on a given input tensor based on the following mathematical formula:
```
[the remark this answers is not in the document]
Comment (ggalieroc) [reply docs/ops/activation/Elu_1.md]: Done

File: docs/ops/activation/Elu_1.md [hunk 2]
```diff
@@ -8,29 +8,58 @@
...
-elu(x) = \left\{\begin{array}{ll}
-    alpha(e^{x} - 1) \quad \mbox{if } x < 0 \\
-    x \quad \mbox{if } x \geq  0
+Elu(x) = \left\{\begin{array}{r}
+    x \qquad \mbox{if } x >  0 \\
+    \alpha(e^{x} - 1) \quad \mbox{if } x \leq 0
 \end{array}\right.
 \f]
 
+Where α, corresponds to *alpha* attribute.
```
[the remark this answers is not in the document]
Comment (ggalieroc) [reply docs/ops/activation/Elu_1.md]: Done

File: docs/ops/activation/Elu_1.md [hunk 3]
```diff
@@ -8,29 +8,58 @@
...
+*Elu* is equivalent to *ReLU* operation when *alpha* is equal to zero. 
+
 **Attributes**
 
 * *alpha*
 
   * **Description**: scale for the negative factor
-  * **Range of values**: arbitrary floating point number
-  * **Type**: float
+  * **Range of values**: non-negative arbitrary floating point number
```
[the remark this answers is not in the document]
Comment (ggalieroc) [reply docs/ops/activation/Elu_1.md]: Done

File: docs/ops/activation/Elu_1.md [hunk 4]
```diff
@@ -8,29 +8,58 @@
...
 
 **Inputs**:
 
-*   **1**: Input tensor x of any floating point type. Required.
+*   **1**: A tensor of type `T` and arbitrary shape. **Required**.
 
 **Outputs**:
 
-*   **1**: Result of Elu function applied to the input tensor *x*. Floating point tensor with shape and type matching the input tensor.
+*   **1**: Result of element-wise *Elu* function applied to the input tensor. A tensor of type `T` and same shape as input tensor.
```
[the remark this answers is not in the document]
Comment (ggalieroc) [reply docs/ops/activation/Elu_1.md]: Done

File: docs/ops/activation/Elu_1.md [hunk 5]
```diff
@@ -8,29 +8,58 @@
...
+
+**Types**
+
+* *T*: arbitrary supported floating point type.
```
[the remark this answers is not in the document]
Comment (ggalieroc) [reply docs/ops/activation/Elu_1.md]: Done

**Pull request merged**